client/src/screens/Patient/Patient.js

```diff
@@ -728,7 +603,7 @@ export default function Patient() {
   const createDocument = (reqBody) => {
     PatientService.createDocuments(patientId, reqBody)
       .then((response) => {
-        dispatch(setSuccess(`${response.data.message}`));
+        reduxDispatch(setSuccess(`${response.data.message}`));
```

Comment:
Use `enqueueSnackbar` for all messages. You can have a look `/screens/patient-portal/auth/Login.js` as an example.

Reply:
Sure I will do that, but there is one thing that needs to be discussed on the call.